webpack/assets/javascripts/react_app/components/toast/toastListitem/Toast.consts.js

```diff
@@ -0,0 +1 @@
+export const defaultTimerDelay = 8000;
```

Comment:
i dont like that defaultProps so much, it is easier to see the const imported from the consts file, then to look for it in the default props value section (that is react dependant, and could get deprecated)